core/players-handbook-2024/classes/class-monk.xml

```diff
@@ -0,0 +1,531 @@
+		<description>
+			<p>Through sheer force of will, you can remove one of the following conditions from yourself at the end of each of your turns: Charmed, Frightened, or Poisoned.</p>
+			<p class="indent">In addition, forgoing food and drink doesn't give you levels of Exhaustion.</p>
+		</description>
+		<sheet alt="Self Restoration">
+			<description>Forgoing food and drink doesn't give you levels of Exhaustion. Additionally, you can remove one of the following conditions from yourself at the end of each of your turns: Charmed, Frightened and Poisoned.</description>
+		</sheet>
+	</element>
+
+
+	<element name="Level 13: Deflect Energy" type="Class Feature" source="Player’s Handbook 2024" id="ID_WOTC_PHB24_CLASS_FEATURE_MONK_DEFLECT_ENERGY">
+		<requirements>!ID_INTERNAL_PHB24_FEATURE_REPLACEMENT_MONK_DEFLECT_ENERGY</requirements>
+		<description>
+			<p>You can now use your Deflect Attacks feature against attacks that deal any damage type, not just Bludgeoning, Piercing, or Slashing.</p>
+		</description>
+		<sheet display="false" />
+	</element>
+
+
+	<element name="Level 14: Disciplined Survivor" type="Class Feature" source="Player’s Handbook 2024" id="ID_WOTC_PHB24_CLASS_FEATURE_MONK_DISCIPLINED_SURVIVOR">
+		<requirements>!ID_INTERNAL_PHB24_FEATURE_REPLACEMENT_MONK_DISCIPLINED_SURVIVOR</requirements>
+		<description>
+			<p>Your physical and mental discipline grant you proficiency in all saving throws.</p>
+			<p class="indent">Additionally, whenever you make a saving throw and fail, you can expend 1 Focus Point to reroll it, and you must use the new roll.</p>
+		</description>
+		<sheet alt="Disciplined Survivor" usage="1 Focus">
+			<description>Whenever you make a saving throw and fail, you can spend 1 Focus Point to reroll it and take the second result.</description>
+		</sheet>
+		<rules>
+			<grant type="Proficiency" id="ID_WOTC_PHB24_PROFICIENCY_SAVINGTHROW_STRENGTH" />
+			<grant type="Proficiency" id="ID_WOTC_PHB24_PROFICIENCY_SAVINGTHROW_DEXTERITY" />
+			<grant type="Proficiency" id="ID_WOTC_PHB24_PROFICIENCY_SAVINGTHROW_CONSTITUTION" />
+			<grant type="Proficiency" id="ID_WOTC_PHB24_PROFICIENCY_SAVINGTHROW_INTELLIGENCE" />
+			<grant type="Proficiency" id="ID_WOTC_PHB24_PROFICIENCY_SAVINGTHROW_WISDOM" />
+			<grant type="Proficiency" id="ID_WOTC_PHB24_PROFICIENCY_SAVINGTHROW_CHARISMA" />
+		</rules>
+	</element>
+
+
+	<element name="Level 15: Perfect Focus" type="Class Feature" source="Player’s Handbook 2024" id="ID_WOTC_PHB24_CLASS_FEATURE_MONK_PERFECT_FOCUS">
+		<requirements>!ID_INTERNAL_PHB24_FEATURE_REPLACEMENT_MONK_PERFECT_FOCUS</requirements>
+		<description>
+			<p>When you roll Initiative and don't use Uncanny Metabolism, you regain expended Focus Points until you have 4 if you have 3 or fewer.</p>
+		</description>
+		<sheet alt="Perfect Focus">
+			<description>When you roll Initiative with 3 or less Focus Points and don't use Uncanny Metabolism, you regain expended Focus Points until you have 4.</description>
```

Comment:
Rare case when you really can't make the description shorter. And while i don't particularly like how it is worded, i think it makes sense to leave as is.
```suggestion
			<description>When you roll Initiative and don't use Uncanny Metabolism, you regain expended Focus Points until you have 4 if you have 3 or fewer.</description>
```